src/models/sequelize/Reply.js

```diff
@@ -1,36 +1,95 @@
 const dbUtils = require('./dbUtils');
 const idGenerator = require('../../services/idGenerator');
 
-/**
- * TODO: implement the rest of Reply model and write tests
- */
-
 module.exports = (sequelize, DataTypes) => {
   const Reply = sequelize.define(
-    'post',
+    'reply',
     {
       id: {
-        type: DataTypes.STRING(60),
+        type: DataTypes.INTEGER,
         defaultValue: () => idGenerator.generate(),
-        primaryKey: true
+        primaryKey: true,
+        allowNull: false
       },
-      authorId: {
-        type: DataTypes.STRING(60),
-        field: 'author_id',
-        index: true,
+      permLink: {
```

Comment:
Why did we remove the foreign key reference to the user that authored the comment? Do sequelize associations add a foreign key automatically?

Reply:
Just adding a comment.  We talked about this after the PR, but just to get it in writing.  We will remove the sequelize associations and manually add in relationships and foreign keys.  This will possibly fix the foreign key constraint error for creating tables.  The other thing that might fix this is doing sequelize migrations,.